examples/experimental/dagster-airlift/dagster_airlift/core/airflow_instance.py

```diff
@@ -265,7 +266,19 @@ def get_dag_runs_batch(
                 f"Failed to fetch dag runs for {dag_ids}. Status code: {response.status_code}, Message: {response.text}"
             )
 
+    @public
     def trigger_dag(self, dag_id: str, logical_date: Optional[datetime.datetime] = None) -> str:
+        """Trigger a dag run for the given dag_id.
+
+        Does not wait for the run to finish. To wait for the completed run to finish, use :py:meth:`wait_for_run_completion`.
+
+        Args:
+            dag_id (str): The dag id to trigger.
+            logical_date (Optional[datetime.datetime]): The logical date to use for the dag run. If not provided, the current time will be used.
```

Comment:
what's a logical date? (if this is a term airflow users will be familiar with, prob no need to add, but as a layperson it is not clear what this indicates)

Reply:
Yea this is an airflow-specific term.